### PR TITLE
fix(docs): changed init subclass

### PR DESCRIPTION
**Pull request by nvy-0x**

### CODE REVIEW

File: eth_portfolio/structs.py [hunk 1]
```diff
@@ -118,11 +118,16 @@ def __init_subclass__(cls, **kwargs):
         # Use the class name exactly as it is defined (e.g., TokenTransfer)
         cls_name = cls.__name__
 
+        # Replace {cls_name} in the class-level docstring
```
Comment (BobTheBuidler) [inline eth_portfolio/structs.py:121]: why are we re-adding this? It doesn't do anything in our use case

File: eth_portfolio/structs.py [hunk 2]
```diff
@@ -118,11 +118,16 @@ def __init_subclass__(cls, **kwargs):
...
+        if cls.__doc__:
+            cls.__doc__ = cls.__doc__.replace("{cls_name}", cls_name)
+
         # Replace {cls_name} in attribute-level docstrings
-        for key in cls.__annotations__:
+        for key, attr_type in cls.__annotations__.items():
```
Comment (BobTheBuidler) [inline eth_portfolio/structs.py:126]: are we using attr_type anywhere?

File: eth_portfolio/structs.py [hunk 3]
```diff
@@ -118,11 +118,16 @@ def __init_subclass__(cls, **kwargs):
...
             attr = getattr(cls, key, None)
-            if attr is not None and attr.__doc__:
-                attr.__doc__ = attr.__doc__.format(cls_name=cls_name)
+            if attr and isinstance(attr, str):  # Ensure that attr is a docstring
```
Comment (BobTheBuidler) [inline eth_portfolio/structs.py:128]: attr will be a function or an attribute, attr.__doc__ would be the docstring for the attr

Comment (BobTheBuidler) [reply eth_portfolio/structs.py:128]: `attr.__doc__` I mean

File: eth_portfolio/structs.py (hunk 1 above)
Comment (nvy-0x) [reply eth_portfolio/structs.py:121]: I think it means the attribute docstrings which are technically a part of the overall class docstrings

File: eth_portfolio/structs.py (hunk 2 above)
Comment (nvy-0x) [reply eth_portfolio/structs.py:126]: changed

File: eth_portfolio/structs.py (hunk 3 above)
Comment (nvy-0x) [reply eth_portfolio/structs.py:128]: see if it works with updated by Cursor

File: eth_portfolio/structs.py (hunk 1 above)
Comment (BobTheBuidler) [reply eth_portfolio/structs.py:120]: but we're doing that immediately below

File: eth_portfolio/structs.py (hunk 3 above)
Comment (BobTheBuidler) [reply eth_portfolio/structs.py]: lol what?

**Pull request merged**